tests/db_tests/test_entities.py

```diff
@@ -97,10 +97,21 @@ def test_database_seed(seeded_app_ctx):
     read_package = Package.query.first()
     assert read_package.package_name == "Regeneration Project"
     assert read_package.fund_type_id == "HIJ"
+
+    # Check fk references
     assert read_package.organisation_id == read_org.id
     assert read_package.cfo_contact_id == read_contact.id
+
+    # Check parent table's row fields available as ORM attributes
     assert read_package.cfo_contact.organisation.organisation_name == "Test Organisation"
 
+    # Check ability to query/read a project
+    read_project = Project.query.first()
+    assert read_project.project_name == "Project 1"
+    assert read_project.address == "SW1A 2AA"
+    assert read_project.secondary_organisation == "Org 1"
+    assert read_project.package_id == read_package.id
+
```

Comment:
Could we add a new test to access the children of Project, rather than just the parents. These are now accessible as ORM attributes directly from project, and should be returned in a list. In your seed data most projects just have one child in each child table so your list should be 1 long. Example implementation:

`assert read_project.outputs[0] == OutputData.query.first()`

Reply:
The reason I've asked for this is that this method of accessing child attributes is specific to the way we've implemented ORM relationships from parent to child. So the test is helpful in case that method gets inadvertently changed.

It may be worth adding it as a completely new test for this reason. The test will probably only have 1 line (ie the example line above) but that's ok - it's just testing the method, not the whole test data integrity.